src/_ert_forward_model_runner/reporting/event.py

```diff
@@ -74,9 +78,9 @@ def __init__(self, evaluator_url, token=None, cert_path=None):
 
         self._ens_id = None
         self._real_id = None
-        self._event_queue = queue.Queue()
+        self._event_queue: queue.Queue[events.Event] | None = queue.Queue()
         self._event_publisher_thread = ErtThread(target=self._event_publisher)
-        self._sentinel = object()  # notifying the queue's ended
+        self._sentinel = None
```

Comment:
just wondering if need this `self._sentinel` object at all. But could be a different PR...